torch/_subclasses/fake_tensor.py

```diff
@@ -821,40 +821,30 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
         # is written to must be invalidated
         self.invalidate_written_to_constants(func, flat_arg_fake_tensors, args, kwargs)
 
-        from torch._decomp import decomposition_table
-
-        with self:
-            # Decomposes CompositeImplicitAutograd ops
-            r = func.decompose(*args, **kwargs)
-            if r is not NotImplemented:
-                return r
+        # If there's a Python meta, prefer that over the decomposition
+        from torch._decomp import meta_table as meta_table
 
-        # IDK: feels bad man, sym_numel on as_strided infinite loops otherwise
-        if has_symbolic_sizes and not self.cpp_meta_supports_symint(func):
```

Comment:
No use of cpp_meta_supports_symint remains. 
we can remove cpp_meta_supports_symint now.